homeassistant/components/tts/microsoft.py

```diff
@@ -37,31 +41,46 @@
 DEFAULT_GENDER = 'Female'
 DEFAULT_TYPE = 'ZiraRUS'
 DEFAULT_OUTPUT = 'audio-16khz-128kbitrate-mono-mp3'
+DEFAULT_RATE = "+0%"
+DEFAULT_VOLUME = "+0%"
+DEFAULT_PITCH = "default"
+DEFAULT_CONTOUR = ""
 
 PLATFORM_SCHEMA = PLATFORM_SCHEMA.extend({
     vol.Required(CONF_API_KEY): cv.string,
     vol.Optional(CONF_LANG, default=DEFAULT_LANG): vol.In(SUPPORTED_LANGUAGES),
     vol.Optional(CONF_GENDER, default=DEFAULT_GENDER): vol.In(GENDERS),
     vol.Optional(CONF_TYPE, default=DEFAULT_TYPE): cv.string,
+    vol.Optional(CONF_RATE, default=DEFAULT_RATE): cv.string,
+    vol.Optional(CONF_VOLUME, default=DEFAULT_VOLUME): cv.string,
```

Comment:
make this as integer as set a range for possible values

Reply:
Same as above.